3.extract_features/pipelines/OP_pipe.cppipe

```diff
@@ -0,0 +1,199 @@
+CellProfiler Pipeline: http://www.cellprofiler.org
+Version:5
+DateRevision:424
+GitHash:
+ModuleCount:13
+HasImagePlaneDetails:False
+
+Images:[module_num:1|svn_version:'Unknown'|variable_revision_number:2|show_window:False|notes:['Add images into this section when using the GUI to test segmentation parameters. Remove prior to exporting as a cppipe file and before using in command line.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
Recommend updating this comment in the GUI to match this pipeline (e.g., removing the statement regarding segmentation since you don't perform segmentation in this pipeline.)